redhat-operators/tektoncd-pipeline-operator/tektoncd-pipeline-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,250 @@
+    mediatype: image/png
+  links:
+  - name: Tekton Pipeline GitHub Repository
+    url: https://github.com/tektoncd/pipeline
+  - name: Tekton Pipeline Operator GitHub Repository
+    url: https://github.com/openshift/tektoncd-pipeline-operator
+  keywords: ['tektoncd', 'build', 'pipeline']
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - name: installs.tekton.dev
+      version: v1alpha1
+      kind: Install
+      displayName: TektonCD-Pipeline Install
+      description: CRD to trigger install/delete of TektonCD-Pipeline
+      resources:
+      - kind: Pod
+        version: v1
+      - kind: Service
+        version: v1
+      - kind: ServiceAccount
+        version: v1
+      - kind: ConfigMap
+        version: v1
+      specDescriptors: []
+      statusDescriptors:
+      - description: List of resources installed by the operator for this cr
+        displayName: Installed Resources
+        path: resources
+      - description: Version of Tektoncd Pipeline installed
+        displayName: Tektoncd Pipeline Version
+        path: version
+  description: Kubernetes operator to install TektonCD-Pipeline
```

Comment:
FYI: This description can be longer and multiline is supported in this case.

Reply:
we have updated it in the latest commit